src/assets/styles/_variables.scss

```diff
@@ -0,0 +1,19 @@
+$spacing-base: 1rem;
+
```

Comment:
This is nice but where did you come from with this idea?
Or just totally made up yourself?

Reply:
I read it once in a book, it reduces the number of spacing values we're using so it makes the interface looks consistent.